.gitignore

```diff
@@ -127,3 +127,4 @@ venv.bak/
 .dmypy.json
 dmypy.json
 
+config.py
```

Comment:
Really minor issue here, `config.py` appears in the file twice.

Reply:
Thanks @janakrajchadha. Removed extra config.py from .gitignore